src/SuperVault.sol

```diff
@@ -790,6 +893,10 @@ contract SuperVault is BaseStrategy, ISuperVault {
                     revert INVALID_SP_FULL_REBALANCE(currentSuperformIds[i]);
                 }
             }
+
+            unchecked {
+                ++i;
+            }
```

Comment:
for loop on line 884: cache length and use `unchecked ++j `

Reply:
We applied this in https://github.com/superform-xyz/SuperVaults/pull/43/commits/ff8833305ff668c1652c0fde574030224430d4e2